testsuite/cluster-test/src/cluster.rs

```diff
@@ -99,17 +113,21 @@ impl Cluster {
         &self.instances
     }
 
-    pub fn prometheus_ip(&self) -> &str {
+    pub fn prometheus_ip(&self) -> &Option<String> {
         &self.prometheus_ip
     }
 
+    pub fn mint_file(&self) -> &str {
+        &self.mint_file
+    }
+
     pub fn get_instance(&self, name: &str) -> Option<&Instance> {
         self.instances
             .iter()
             .find(|instance| instance.short_hash() == name)
     }
 
-    pub fn sub_cluster(&self, ids: Vec<String>) -> Cluster {
+    pub fn sub_cluster(&self, ids: Vec<String>, mint_file: &str) -> Cluster {
```

Comment:
I don't think we have use case for getting subcluster with different mint file, we can just copy value from parent cluster...